safe_transaction_service/history/cache.py

```diff
@@ -96,7 +96,9 @@ def remove_cache(self):
 
 
 def cache_txs_view_for_address(
-    cache_tag: str, timeout: int = settings.CACHE_VIEW_DEFAULT_TIMEOUT
+    cache_tag: str,
+    parameter_key: Optional["str"] = "address",
```

Comment:
Parameter is not optional

```suggestion
    parameter_key: str = "address",
```